crates/bitwarden/src/mobile/vault/folders/mod.rs

```diff
@@ -0,0 +1,42 @@
+use chrono::{DateTime, Utc};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::{
+    client::encryption_settings::EncryptionSettings,
+    crypto::{CipherString, Decryptable},
+    error::Result,
+};
+
+mod request;
+mod response;
+
+pub use request::*;
+pub use response::*;
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct Folder {
+    id: Uuid,
+    name: CipherString,
+    revision_date: DateTime<Utc>,
+}
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct FolderView {
+    id: Uuid,
+    name: String,
+    revision_date: DateTime<Utc>,
+}
```

Comment:
I would like to move these up into a root `vault` module.

Reply:
So just in a `mobile/vault/folders.rs` file, and keeping the `folders` folder for everything else?